xtext-website/community.html

```diff
@@ -1094,6 +1094,14 @@ <h2>What others have built with Xtext</h2>
 						<td>Lorenzo Bettini</td>
 					</tr>
 
+					<tr>
+						<td><a href="https://github.com/Yakindu/solidity-ide">YAKINDU Solidity IDE</a></td>
+						<td>The free to use, open source YAKINDU Solidity IDE provides an Xtext-based integrated development environment for Ethereum / Solidity-based smart contracts. It can be installed in different environments, such as Atom, Eclipse, Theia and VSCode. <a href="https://solidity-ide.itemis.de/">Online editor</a> and <a href="https://yakindu.github.io/solidity-ide/">documentation</a>.</td>
```

Comment:
the sentence ending with documentation makes no sense

Reply:
I took that from the description of the Balzac project:
![screenshot](https://user-images.githubusercontent.com/20393472/84737839-06677a00-afa9-11ea-9cec-12b316053d64.png)
I think it is not so bad.